API Center SDK Update

alexyaang opened the pull request:
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

yangalex1 committed:
Added specs

yangalex1 committed:
Added specs

yangalex1 committed:
Update TagPrefix in assets.json

yangalex1 committed:
Update TagPrefix in assets.json

yangalex1 committed:
Refactor debug logging in Export-API.ps1 script

yangalex1 committed:
Refactor debug logging in Export-API.ps1 script

yangalex1 committed:
Update model serialization to use System.ClientModel

yangalex1 committed:
Update model serialization to use System.ClientModel

yangalex1 committed:
Fix Managed Service Identity configuration

yangalex1 committed:
Fix Managed Service Identity configuration

ArthurMa1978 committed:
Merge branch 'main' into apicenter

ArthurMa1978 committed:
update code

ArthurMa1978 committed:
update code

ArthurMa1978 commented on sdk/apicenter/Azure.ResourceManager.ApiCenter/CHANGELOG.md:
```diff
@@ -1,17 +1,11 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.3 (2024-04-09)
```
Can this be stable as the swager spec is stable version.

alexyaang replied:
@azaslonov thoughts?

azaslonov replied:
Yes please, it should be stable 1.0.0 version. @alexyaang could you please change?

alexyaang replied:
changed it to `1.0.0`, good catch guys

jsquire reviewed (changes requested):
@alexyaang :

You are missing the needed permissions and public group memberships for an internal contributor.  Please review the Azure SDK onboarding documentation and use the onboarding teams channel for any questions.
- [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
- [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_

Please also be sure to add yourself to [CODEOWNERS](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS) for this library, if you will be maintaining it going forward.

yangalex1 committed:
Add code owners for API Center

yangalex1 committed:
Add code owners for API Center

yangalex1 committed:
Merge branch 'main' of https://github.com/alexyaang/azure-sdk-for-net

alexyaang commented:
@jsquire my apologies for misreading the onboarding guide. I have adjusted my `@Azure` org membership visibility, requested for permission to join the partner group (still pending approval), and added me and @azaslonov to the owners file as owners of `sdk/apicenter`. That seems to be all from the documentation, but please let me know if I'm missing something.

yangalex1 committed:
Update CODEOWNERS file with new service owners

yangalex1 committed:
Update CODEOWNERS file with new service owners

yangalex1 committed:
Update version number in CHANGELOG.md

yangalex1 committed:
Update version number in CHANGELOG.md

alexyaang commented:
@jsquire Update: I have joined the partner group & adjusted my @Azure org membership visibility. 

I requested review from you

jsquire commented on .github/CODEOWNERS:
```diff
@@ -228,6 +228,12 @@
 /sdk/anomalydetector/                                              @conhua @mengaims @juaduan @moreOver0
 
+# PRLabel: %API Center
+/sdk/apicenter/                                                    @alexyaang @azaslonov
```
This needs to move down to the management section, around L930.   Please organize to preserve alphabetical ordering.

alexyaang replied:
I think I get what you mean? I made a commit here https://github.com/Azure/azure-sdk-for-net/pull/43013/commits/af80885324bea1030fecb2d186f2aded9bd247f5, is this what you meant?

jsquire commented on .github/CODEOWNERS:
```diff
@@ -228,6 +228,12 @@
 /sdk/anomalydetector/                                              @conhua @mengaims @juaduan @moreOver0
 
+# PRLabel: %API Center
+/sdk/apicenter/                                                    @alexyaang @azaslonov
```
```suggestion
/sdk/apicenter/Azure.ResourceManager.ApiCenter/                     @alexyaang @azaslonov
```

jsquire commented on .github/CODEOWNERS:
```diff
@@ -228,6 +228,12 @@
+# PRLabel: %API Center
+/sdk/apicenter/                                                    @alexyaang @azaslonov
+
+# ServiceLabel: %API Center
```
```suggestion
# ServiceLabel: %API Center %Mgmt
```

jsquire reviewed (dismissed):
@alexyaang:

Your account still lacks the needed write permissions of a Microsoft contributor/Code owner.

alexyaang committed:
Update .github/CODEOWNERS

Co-authored-by: Jesse Squire <jesse.squire@gmail.com>

alexyaang committed:
Update .github/CODEOWNERS

Co-authored-by: Jesse Squire <jesse.squire@gmail.com>

alexyaang committed:
Update .github/CODEOWNERS

Co-authored-by: Jesse Squire <jesse.squire@gmail.com>

alexyaang committed:
Update .github/CODEOWNERS

Co-authored-by: Jesse Squire <jesse.squire@gmail.com>

yangalex1 committed:
Update CODEOWNERS file

yangalex1 committed:
Update CODEOWNERS file

ArthurMa1978 committed:
Merge branch 'main' into apicenter

ArthurMa1978 committed:
Update

ArthurMa1978 committed:
Update

ArthurMa1978 committed:
update

ArthurMa1978 committed:
update

alexyaang commented:
@ArthurMa1978 I believe all this PR needs is management approval from you, just an FYI. Thanks in advance

alexyaang commented:
> @alexyaang: Please do not ever remove the review of an Azure SDK team member without their explicit consent.

Sorry, not sure what you're referring to, what did I do exactly?

alexyaang commented:
@jsquire Ah okay, sorry didn't mean to do that. I thought it was equivalent of resolving a comment

ArthurMa1978 committed:
Merge branch 'main' of https://github.com/alexyaang/azure-sdk-for-net into apicenter

ArthurMa1978 committed:
Merge branch 'main' into apicenter

ArthurMa1978 commented on sdk/apicenter/Azure.ResourceManager.ApiCenter/assets.json:
```diff
@@ -1,6 +1,6 @@
   "AssetsRepo": "Azure/azure-sdk-assets",
   "AssetsRepoPrefixPath": "net",
-  "TagPrefix": "net/<service name>/Azure.ResourceManager.ApiCenter",
+  "TagPrefix": "net/apicenter/Azure.ResourceManager.ApiCenter",
```
Since this is stable release please follow
[azure-sdk-for-net/sdk/resourcemanager/Azure.ResourceManager/docs/TestGuide.md at main · Azure/azure-sdk-for-net (github.com)](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/docs/TestGuide.md)
to add tests and record the test result
